### PR TITLE
chore: isSupportedFormat ts ext name

### PR DESCRIPTION
**Pull request by ErKeLost**
### Test Plan

---

### CODE REVIEW

File: packages/rolldown/src/cli/utils.ts [hunk 1]
```diff
@@ -23,5 +23,5 @@ export async function loadConfig(
  */
 function isSupportedFormat(configPath: string): boolean {
   const ext = path.extname(configPath)
-  return /\.(js|mjs)$/.test(ext)
+  return /\.(js|mjs|ts)$/.test(ext)
```
Comment (kazupon) [inline packages/rolldown/src/cli/utils.ts]: As commented [here](https://github.com/rolldown/rolldown/pull/610#issuecomment-2009485012), we are currently not supporting the ts extension.

Comment (ErKeLost) [reply packages/rolldown/src/cli/utils.ts]: Thank you for your reply. I see that error's prompt supports ts format, so do you think other logic optimizations can be merged? I can remove the rules of this ts type.

**Review by kazupon — APPROVED** [review]
Your PR looks a good refactoring for me. :)

**Review by hyf0 — APPROVED** [review]
Thanks!

**Pull request merged**